public/index.ts

```diff
@@ -1094,7 +1121,8 @@ async function loadTestData(testdata: TestDataSpec) {
 
   const loadSpec = new LoadSpec();
   myState.totalFrames = testdata.times;
-  loadVolume(loadSpec, myState.loader);
+  const loader = Array.isArray(myState.loader) ? myState.loader[myState.scene] : myState.loader;
+  loadVolume(loadSpec, loader);
```

Comment:
I'm not sure I'm following the full data flow yet (just started reading through this). but I'm seeing a lot of `isArray` checks because it may be a single scene or multiple scenes. But it seems like you should be able to convert the data to an array on entry and just sometimes deal with an array of length 1? I just think it's cleaner to have a consistent data type.